universal-login-sdk/lib/sdk.js

```diff
@@ -93,14 +89,17 @@ class EthereumIdentitySDK {
     throw new Error(`${response.status}`);
   }
 
-  async execute(contractAddress, message, privateKey) {
+  async execute(message, privateKey) {
     const url = `${this.relayerUrl}/identity/execution`;
     const method = 'POST';
-    const wallet = new ethers.Wallet(privateKey, this.provider);
-    const nonce = parseInt(await this.getNonce(contractAddress, wallet), 10);
-    message.nonce = nonce;
-    const signature = calculateMessageSignature(privateKey, {...message, from: contractAddress});
-    const body = JSON.stringify({...this.defaultPaymentOptions, ...message, contractAddress, signature});
+    const msg = {
```

Comment:
such an ugly short... why not `finalMessage`

Reply:
✅